.github/scripts/pack_macos.sh

```diff
@@ -44,11 +44,16 @@ notarize_and_staple() {
 }
 
 pack_macos() {
+  if ["$REPOSITORY_OWNER" == "twilio"]
+  then
     import_certificate
     npx oclif-dev pack:macos
     notarize_and_staple
+  else
+    npx oclif-dev pack:macos
```

Comment:
This still might not work on forked repos, Apple needs its package to be signed before releasing. The operation could fail. Any workflow run to verify this?

Reply:
https://github.com/charan678/twilio-cli/runs/5124546146?check_suite_focus=true